addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -2246,7 +2246,7 @@ export class OdooEditor extends EventTarget {
     _handleSelectionInTable(ev=undefined) {
         const selection = this.document.getSelection();
         // Selection could be gone if the document comes from an iframe that has been removed.
-        const anchorNode = selection && selection.getRangeAt(0) && selection.anchorNode;
+        const anchorNode = selection.rangeCount && selection.getRangeAt(0) && selection.anchorNode;
```

Comment:
@dmo-odoo @Goaman Since you added an explicit check to ensure `selection` is not falsy and that BVR is now removing it, could you be the ones that merge this PR?

@bvr-odoo Thanks for the quick fix. Maybe:
- Why is the <main> element even activable at all? I guess it's unrelated to this editor lib crash but why does it get a blue overlay with no option at all?
- Possible to add a test about this? Although, quite annoying that the error does not seem systematic... (why is it not systematic, could there be something else to improve?)

Reply:
@bvr-odoo Could you also always remove the "Description of the issue/feature this PR addresses:" etc part of the PR description? :disappointed:

Reply:
> @dmo-odoo @Goaman Since you added an explicit check to ensure `selection` is not falsy and that BVR is now removing it, could you be the ones that merge this PR?

Ah yes you're right. Good catch. It should be `selection && selection.rangeCount` indeed. In master we'd have done `selection?.rangeCount` but the future isn't now yet.

We can indeed merge the PR.

Reply:
@dmo-odoo Just curious: in which case is there no selection?

Reply:
When the node in which the selection was has been deleted, or the focus has gone to a node which cannot be selected per se. What makes it more likely that it seems is that the `selectionchange` event is async, meaning it's possible that the node in which the selection was has already been deleted by the time it is triggered.

However, you are right that neither of those cases are properly covered by the actual code, in fact.

In OdooEditor.js `_onSelectionChange`: 💥 
```js
const selection = this.document.getSelection();
const anchorNode = selection.anchorNode;
```

In OdooEditor.js `_onMouseMove`: 💥 
```js
if (this._currentMouseState === 'mousedown' && !this._isResizingTable) {
    this._handleSelectionInTable(ev);
}
```

I guess it's the second one that is triggering the issue here.